virttest/qemu_storage.py

```diff
@@ -823,6 +829,69 @@ def _backing_access_secret_objects(self):
 
         return secrets
 
+    def _is_legacy_qemu(self):
+        """
+        Check if this is a legacy QEMU version (before 10.1.0).
+
+        :return: True if QEMU version is before 10.1.0, False otherwise
+        :rtype: bool
+        """
+        return self.qemu_ver in VersionInterval("(,10.1.0)")
+
+    def _get_cmd_help_text(self, cmd):
```

Comment:
It might be better to move this function to a utils module, but currently we don't have a utils_qemu_img module yet. We have a utils_qemu module which is only used for qemu. Any thoughts?